GPU/D3D11/DrawEngineD3D11.cpp

```diff
@@ -111,18 +112,18 @@ void DrawEngineD3D11::InitDeviceObjects() {
 }
 
 void DrawEngineD3D11::ClearTrackedVertexArrays() {
-	for (auto &vai : vai_) {
-		delete vai.second;
-	}
-	vai_.clear();
+	vai_.Iterate([&](uint32_t hash, VertexArrayInfoD3D11 *vai){
```

Comment:
I wonder if this is common enough we just have a specialized PointerDenseMap or something with a DeleteAll?  I suppose we have Release and other variants so maybe not...

-[Unknown]